### PR TITLE
update docs on mgmt namespace

### PR DESCRIPTION
**Pull request by m-nash**

### CODE REVIEW

File: docs/dotnet/introduction.md [hunk 1]
```diff
@@ -810,6 +810,8 @@ For example, `Azure.Storage.Blobs`.
 - `Azure.Management` for client libraries accessing the control plane (Azure Resource Manager)
 - `Azure.Media` for client libraries that deal with audio, video, or mixed reality
 - `Azure.Messaging` for client libraries that provide messaging services, such as push notifications or pub-sub.
+- `Azure.ResourceManager.[ResourceProvider]` for management plane client libraries for a given resource provider.
```
Comment (tg-msft) [inline docs/dotnet/introduction.md:813]: You should remove the outdated `Azure.Management` above.

Comment (tg-msft) [reply docs/dotnet/introduction.md:813]: Should we offer any guidance on how to name the resource provider part of the namespace (i.e., how do I turn https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers into a namespace name)?

File: docs/dotnet/introduction.md [hunk 2]
```diff
@@ -810,6 +810,8 @@ For example, `Azure.Storage.Blobs`.
...
+    - For example the compute managemenat plane namespace would be Azure.ResourceManager.Compute.
```
Comment (tg-msft) [inline docs/dotnet/introduction.md:814]:
```suggestion
    - For example the compute management plane namespace would be Azure.ResourceManager.Compute.
```

**Review by tg-msft — COMMENTED** [review]
I'm okay with this, but I want @KrzysztofCwalina to do the approval.

File: docs/dotnet/introduction.md (hunk 1 above)
Comment (allenjzhang) [reply docs/dotnet/introduction.md:813]: Most existing RP would have the name established. For new ones, Archboard review is required.

**Review by KrzysztofCwalina — APPROVED** [review]

**Pull request merged**